hack/lib/build/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This library holds utility functions for building releases.
+
+# os::build::release::check_for_rpms checks that an RPM release has been built
+function os::build::release::check_for_rpms() {
+	if [[ ! -d "${OS_LOCAL_RPMPATH}" || ! -s "${OS_LOCAL_RELEASEPATH}/CHECKSUM" ]]; then
```

Comment:
Yeah, now I see. For some reason I am not getting back from `hack/env` anything related to the release binaries.. Looking further..